src/engraving/rendering/dev/systemlayout.cpp

```diff
@@ -992,7 +992,7 @@ void SystemLayout::layoutSystemElements(System* system, LayoutContext& ctx)
         }
         staff_idx_t si = e->staffIdx();
         EngravingItem* parent = e->parentItem(true);
-        IF_ASSERT_FAILED(parent && parent->isSegment());
+        IF_ASSERT_FAILED(parent && parent->isSegment())
```

Comment:
But why that empty `IF_ASSERT_FAILED()` in the first place, when `toSegment(parent)` does the very same?
@mike-spa ?